src/framework/_layers.scss

```diff
@@ -30,3 +32,18 @@ $layers: (
   $value: map-get($layers, $layer);
   @return $value + $var;
 }
+
+@if $has-classes {
+	@each $layer in $layers {
+    $name: nth($layer, 1);
+    $value: z($name);
+
+    .layer-#{$name} {
+      z-index: $value;
+    }
+
+    .layer-#{$name}--force {
+      z-index: $value !important;
+    }
+  }
```

Comment:
👍 
Do we also need a per breakpoint class ? 
`.layer-goku--s` ?

Reply:
It's now implemented
`.layer-goku--xxs`
`.layer-goku--force-xxl`